qutip/about.py

```diff
@@ -61,6 +63,19 @@ def about():
     qutip_install_path = os.path.dirname(inspect.getsourcefile(qutip))
     print("Installation path:  %s" % qutip_install_path)
 
+    # family packages
+    entrypoints = pkg_resources.iter_entry_points('qutip.about')
```

Comment:
It is better to use importlib rather than `pkg_resources` as stated in their [documentation](https://setuptools.pypa.io/en/latest/pkg_resources.html). The latter might be deprecated in the future.